components/mindstorms-ev3/src/site/markdown/index.md

```diff
@@ -0,0 +1,3 @@
+# EV3

```

Comment:
Should this be filled in with something?

Reply:
I don't actually know. Is this supposed to be the same as the readme?

Reply:
On some other community components it looks like it is, but on others, it isn't. So I'll say yes (@TIBCOjstinton ?)

Reply:
I don't think there is a good convention for this yet.  We use the README.md in the root of the project for all of our links and the sample importer to load by default. The /src/site/markdown folder and files are generated by default by our maven archetype.  I think leaving it with default comments, or simply deleting the /src/site folder in general are fine.  Just make sure it hasn't been copy/pasted from another project and therefore has incorrect data in it.